src/scss/login.module.scss

```diff
@@ -252,7 +256,7 @@ $white: #fff;
 %button,
 .button {
   padding: 2px 8px 0 10px;
-  border-radius: 5px;
+  
```

Comment:
nit: A lot of extra lines seems to have been committed, could you run the file through a basic formatter?